src/ProjectTemplates/ComponentsWebAssembly.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Server/Properties/launchSettings.json

```diff
@@ -28,6 +29,7 @@
         //#else
         "applicationUrl": "http://localhost:5000",
         //#endif
+        "inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}",
```

Comment:
I'm not sure why `inspectUri` goes *before* `applicationUrl` in the "client" project, but *after* it in the "server" project.

It won't actually cause any problems either way but we usually try to keep things like this aligned just because it makes source control diffs and copy-paste style upgrades easier to reason about.

Will still approve the PR. Leaving it up to you to determine whether to change this!